cpu/atxmega/Makefile.features

```diff
@@ -5,10 +5,5 @@ include $(RIOTCPU)/avr8_common/Makefile.features
 
 FEATURES_PROVIDED += cpu_core_atxmega
 FEATURES_PROVIDED += periph_cpuid
-FEATURES_PROVIDED += periph_gpio periph_gpio_irq
-FEATURES_PROVIDED += periph_i2c
 FEATURES_PROVIDED += periph_nvm
 FEATURES_PROVIDED += periph_pm
-FEATURES_PROVIDED += periph_spi
-FEATURES_PROVIDED += periph_timer periph_timer_periodic
```

Comment:
`periph_timer_periodic` should stay a CPU feature as it's more a hint of that the implementation support that feature - it does not require any board specific configuration.